opifancontrol.sh

```diff
@@ -28,7 +28,7 @@ CURRENT_PWM=0
 LAST_RAMPED_DOWN_TS=0
 
 # Initialize GPIO pin for PWM
-gpio export $FAN_GPIO_PIN out
```

Comment:
The purpose is not clear based on https://github.com/orangepi-xunlong/wiringOP/blob/39cb92098ddd3e70eb7424793151c02864a34e29/gpio/gpio.1#L203 - likely it does not do, what it was supposed to.